config/database.yml

```
@@ -14,7 +14,7 @@ default: &default
   encoding: utf8
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
   username: root
-  password: mysecretpassword
+  password: <%= ENV['DISCOVERY_DATABASE_PASSWORD'] %>
```

I'm going to open another issue to talk about fixing this to be in-line with Jupiter.  Could we add the old value as a default to allow backward compatible for my setup?
```suggestion
  password: <%= ENV['DISCOVERY_DATABASE_PASSWORD'] || 'mysecretpassword' %> 
```